clients/filter/filter.go

```diff
@@ -92,11 +89,16 @@ func (c *Client) GetOutput(ctx context.Context, filterOutputID string) (m Model,
 		return
 	}
 
+	common.AddServiceTokenHeader(req, authToken)
+	common.AddDownloadServiceTokenHeader(req, downloadServiceToken)
+
 	resp, err := c.cli.Do(ctx, req)
 	if err != nil {
 		return
 	}
 
+	defer resp.Body.Close()
```

Comment:
Another minor point but `Close` can return an error which we often don't bother to handle (again not something you've specifically done - something littered all across our apps).

I would suggest deferring a call to a close response func which logs the error:

```
func closeResponseBody(ctx context.Context, resp *http.Response) {
	if err := resp.Body.Close(); err != nil {
		log.Event(ctx, "error closing http response body", log.Error(err))
	}
}
```

```
defer closeResponseBody(ctx, resp) 
```

Reply:
Are we wanting to change the logging library for this whilst we are at it? As `clientlog` uses the go-ns logs. So by changing the logs of this file to the `log.go` logs used in the example above we can't give clientlog `log.Data` anymore. 

Alternatively, we can cast from one log form to the other and keep the old log import in the file and add a new one, but call the old one by a new keyword e.g. legacyLog instead (Not great keeping both logs, but a stepping stone in replacing the old one).

If neither then will use the old logging system for the example above.

My vote would be for us to replace the logging system but would need to agree as a team I think (as it is a bit of an increase in feature scope).